app/pkg/dbx/migrate.go

```diff
@@ -140,3 +144,27 @@ func getLastMigration() (int, error) {
 
 	return int(lastVersion.Int64), nil
 }
+
+func getPendingMigrations(versions []int) ([]int, error) {
+	versionStr := ""
+	for _, version := range versions {
+		versionStr = versionStr + "," + strconv.Itoa(version)
+	}
+	versionStr = versionStr[1:]
```

Comment:
If you parametize the query, you can then pass the array directly to the conn.Query call, you shouldn't need to convert it to a CSV like this. Something like

```
    rows, err := conn.Query("SELECT version FROM migrations_history WHERE version = ANY($1)", pq.Array(versions))
```

Reply:
Much more readable like this, I didn't know parametized queries with Go 👍